neverbleed.h

```diff
@@ -106,14 +106,15 @@ extern void (*neverbleed_post_fork_cb)(void);
  * The callback returns a boolean indicating if it handled the task. It may return false to delagate the task back to the default
  * logic.
  */
-extern void (*neverbleed_transaction_cb)(neverbleed_iobuf_t *);
+extern void (*neverbleed_transaction_cb)(neverbleed_iobuf_t *, int);
 
 typedef void (*neverbleed_cb)(int);
 
 int neverbleed_get_fd(neverbleed_t *nb);
 static size_t neverbleed_iobuf_size(neverbleed_iobuf_t *buf);
 void neverbleed_transaction_read(neverbleed_t *nb, neverbleed_iobuf_t *buf);
 void neverbleed_transaction_write(neverbleed_t *nb, neverbleed_iobuf_t *buf);
+void neverbleed_transaction_dispose(neverbleed_iobuf_t *buf);
```

Comment:
PS. I think it makes sense to add a public function for disposing `neverbleed_iobuf_t` but if we are to add,
* I think it should be named `neverbleed_iobuf_dispose` as it deals with `neverbleed_iobuf_t`. We have `neverbleed_iobuf_size` as the precedent.
* We should use the newly introduced function in `async_nb_job_free` as well.

The alternative would be continue freeing `buf->buf` directly.